src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -101,7 +127,23 @@ export const _Chatbox: React.FC<ChatboxProps> = ({
 
   return (
     <div className="Chatbox">
-      <div className="Chatbox__messages">{renderedMessages}</div>
+      <div
+        className="Chatbox__messages-container"
+        ref={scrollableComponentRef}
+        id={"Chatbox_scrollable_div"}
+      >
+        <InfiniteScroll
+          dataLength={messagesRenderCount}
+          className={"Chatbox__messages-infinite-scroll"}
+          next={renderMore}
+          inverse={true}
+          hasMore={messagesRenderCount < messages.length}
+          scrollableTarget="Chatbox_scrollable_div"
+          loader={<h4>Loading...</h4>}
```

Comment:
Use this component as a loader please `import { Loading } from "components/molecules/Loading";`

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/30d70804f78750583ccede9163b98938e4ea61f8